server/cmd/mmctl/client/client.go

```diff
@@ -127,10 +127,11 @@ type Client interface {
 	UploadData(ctx context.Context, uploadID string, data io.Reader) (*model.FileInfo, *model.Response, error)
 	ListImports(ctx context.Context) ([]string, *model.Response, error)
 	GetJob(ctx context.Context, id string) (*model.Job, *model.Response, error)
-	GetJobs(ctx context.Context, page int, perPage int) ([]*model.Job, *model.Response, error)
+	GetJobs(ctx context.Context, page int, perPage int, jobType string, status string) ([]*model.Job, *model.Response, error)
```

Comment:
Other API method put the options first, so
```suggestion
	GetJobs(ctx context.Context,  jobType, status string, page int, perPage int) ([]*model.Job, *model.Response, error)
```

Reply:
Done